static/js/containers/App.js

```diff
@@ -79,6 +80,11 @@ export class App extends React.Component<Props, void> {
             path={urljoin(match.url, routes.dashboard)}
             component={DashboardPage}
           />
+          <PrivateRoute
+            exact
```

Comment:
do we really need exact here? as we don't have Receipt list page so it would only some record_id.

Reply:
Removed